backend/frontend/views.py

```diff
@@ -294,6 +294,24 @@ def action_start_card(request, card_id):
         },
     )
 
+@user_passes_test(is_super)
```

Comment:
Percy will be able to help you with the permissions. He wrote a bunch of stuff. 
We will need anyone with view permission to see the card details.